src/textord/tordmain.cpp

```diff
@@ -533,16 +533,17 @@ bool Textord::clean_noise_from_row( // remove empties
       }
     }
   }
+  bool rejected = dot_count > norm_count * textord_noise_normratio && \
+                  dot_count > 2 && \
+                  super_norm_count < textord_noise_sncount;
   if (textord_noise_debug) {
     tprintf("Row ending at (%d,%g):", blob_box.right(), row->base_line(blob_box.right()));
     tprintf(" R=%g, dc=%d, nc=%d, %s\n",
             norm_count > 0 ? static_cast<float>(dot_count) / norm_count : 9999, dot_count,
             norm_count,
-            dot_count > norm_count * textord_noise_normratio && dot_count > 2 ? "REJECTED"
-                                                                              : "ACCEPTED");
+            rejected? "REJECTED": "ACCEPTED");
```

Comment:
The original code did not use `super_norm_count < textord_noise_sncount` here.

Reply:
But it includes the expression in the return statement.

Reply:
```python
return super_norm_count < textord_noise_sncount &&
         dot_count > norm_count * textord_noise_rowratio && dot_count > 2;
```

Reply:
Yes, the debug message and the return statement used different expressions in the past. I don't know whether that was intentional or a bug. The new code uses the same expression for both locations.